api/capi/src/nnstreamer-capi-single.c

```diff
@@ -310,6 +310,7 @@ ml_single_set_gst_info (ml_single * single_h, const ml_tensors_info_h info)
   int ret = -EINVAL;
 
   ml_tensors_info_copy_from_ml (&gst_in_info, info);
+  gst_tensors_info_init (&gst_out_info);
```

Comment:
plz remove this line. (ml_tensors_info_copy_from_ml() includes gst_tensors_info_init)
This will init all gst-info, unexpected.

Reply:
Oh...it's my mistake.